src/OrbitQt/ProfilingTargetDialog.ui

```diff
@@ -182,6 +182,10 @@
               <property name="text">
                <string/>
               </property>
+              <!-- TODO (b/178379643): As part of rethinking the load capture flow, decide if this label is necessary -->
```

Comment:
Is this todo preserved by designer changes? If not, is it important enough to impose the manual step of re-adding the comment whenever someones changes this UI file in the designer?

Reply:
You are right, they are not preserved between saves in designer. I removed the comment and added a note about this in b/178379643 instead. Thanks for catching that!